server/modules/tasks/server/models/sequelize/comments.server.model.sequelize.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+
+module.exports = function(sequelize, DataTypes) {
```

Comment:
It's a sequelize model and we are instantiating them automatically from `config/lib/sequelize.js` where they get 'required' and called with those arguments.